util/proposal.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"regexp"
+
+	www "github.com/decred/politeia/politeiawww/api/v1"
+)
+
+var (
+	validProposalName = regexp.MustCompile(www.ValidProposalNameRegExp)
+)
+
+// ProposalName returns a proposal name
+func ProposalName(payload string) (string, error) {
```

Comment:
I'm not crazy about this function name, shouldn't it have a verb, or is this standard for Go functions?